src/main/features/claimant-response/views/macro/termsOfAgreementTable.njk

```diff
@@ -0,0 +1,34 @@
+{% macro termsOfAgreement(id, response, paymentPlan) %}
+  <fieldset id="{{ id }}" class="form-group">
+    <legend>
+      <span class="visually-hidden">{{ t( 'Payment plan' ) }}</span>
+    </legend>
+
+    {% if response.paymentIntention.paymentDate %}
+      {{ row('The offer', t('{{ defendantFullName }} will pay the full amount, no later than {{ paymentDate }}',
+        { defendantFullName: response.defendant.name, paymentDate: response.paymentIntention.paymentDate | date }
+      )) }}
+      {{ row('Completion date', response.paymentIntention.paymentDate | date) }}
+    {% elseif response.paymentIntention.repaymentPlan %}
+      {{ row('The offer', t('{{ defendantFullName }} will pay instalments of {{ instalmentAmount }} every month. The first instalment will be paid by {{ firstPaymentDate }}.',
+        { defendantFullName: response.defendant.name,
+          instalmentAmount: response.paymentIntention.repaymentPlan.instalmentAmount | numeral,
```

Comment:
I doubt this is correct. A `response.paymentIntention.repaymentPlan.instalmentAmount` does not mean amount will be paid every month. It depends on frequency, that could be every week or every two weeks.